fastlane/lib/fastlane/actions/delete_keychain.rb

```diff
@@ -4,25 +4,17 @@ module Fastlane
   module Actions
     class DeleteKeychainAction < Action
       def self.run(params)
-        original = Actions.lane_context[Actions::SharedValues::ORIGINAL_DEFAULT_KEYCHAIN]
-
-        search_paths = []
-        search_paths << File.expand_path(params[:name]) if params[:name]
-        search_paths << File.expand_path(File.join("~", "Library", "Keychains", params[:name])) if params[:name]
-        search_paths << File.expand_path(params[:keychain_path]) if params[:keychain_path]
```

Comment:
For reference, [here's the relevant code](https://github.com/fastlane/fastlane/blob/f7a5071122aad789d7c1451509cbbd1efea494e8/fastlane_core/lib/fastlane_core/helper.rb#L203) in the helper method